packages/lodestar/src/opPool/opPool.ts

```diff
@@ -74,4 +76,39 @@ export class OpPool extends EventEmitter {
     ];
     await Promise.all(tasks);
   }
+
+  public async checkDuplicateProposer(config, block: BeaconBlock): Promise<void> {
+    const epoch: Epoch = computeEpochOfSlot(config, block.slot);
+    const proposers: Map<ValidatorIndex, Slot> = this.proposers.get(epoch);
+    const proposerIndex: ValidatorIndex = await this.db.getValidatorIndex(block.signature);
```

Comment:
the param for `db.getValidatorIndex` is a public key, not a signature
I think you should use `getBeaconProposerIndex` to get the proposer index.

Reply:
you should call getBeaconProposerIndex with block slot to get validator index